docs/warehouse/developing_dbt_models.md

```diff
@@ -80,7 +84,8 @@ tests_and_docs --> merge_model_changes
 ```
 
 (identify-bug)=
-### Identify the cause of your bug.
+
+### Identify the cause of your bug
```

Comment:
if you are deleting the periods in these section headers can you delete them in the Mermaid diagram too? goal was consistency. Not a huge deal but if we are doing this kind of editing may as well keep it consistent if we can

Reply:
Resolved in [e2ff4ca](https://github.com/cal-itp/data-infra/pull/2896/commits/e2ff4cafe8678fbd6bcfa2a82c4d9124d6b4cef2)